src/common/components/datepicker/__tests__/Datepicker.test.tsx

```diff
@@ -54,24 +53,29 @@ const pressKey = ({ key, keyCode }: { key: string; keyCode?: number }) => {
 
 advanceTo('2020-07-05');
 
+const getElement = (key: 'toggleButton') => {
+  switch (key) {
+    case 'toggleButton':
+      return screen.getByRole('button', { name: /valitse päivämäärä/i });
+  }
+};
+
 describe('<Datepicker />', () => {
   it('Datepicker opens when user focuses with tab', () => {
```

Comment:
Is this test description outdated?

Reply:
In fact, the whole test is out of date. Same thing is tested in `should toggle datepicker by clicking toggle button`. Removed the test as redundant